Neos.ContentRepository.TestSuite/Classes/Behavior/Features/Bootstrap/CRBehavioralTestsSubjectProvider.php

```diff
@@ -181,22 +181,31 @@ protected function setUpContentRepository(ContentRepositoryId $contentRepository
          * Catch Up process and the testcase reset.
          */
         $contentRepository = $this->createContentRepository($contentRepositoryId);
-        $contentRepositoryMaintainer = $this->contentRepositoryRegistry->buildService($contentRepositoryId, new ContentRepositoryMaintainerFactory());
```

Comment:
we are not allowed to access `contentRepositoryRegistry` as this is not even declared in this trait and just worked by chance ... the test suite is independent of the registry